packages/core/src/chatgpt/ChatGptInitializeFunctionAgent.ts

```diff
@@ -61,12 +71,14 @@ export namespace ChatGptInitializeFunctionAgent {
       if (
         choice.message.role === "assistant" &&
         !!choice.message.content?.length
-      )
+      ) {
+        // @TODO this logic should call the dispatch function
```

Comment:
initialize text dispatch is not called
Is there a particular reason? @samchon

Reply:
<img width="443" alt="image" src="https://github.com/user-attachments/assets/73860d1c-1be1-4d5d-9929-3bcb4d2b570a" />

No such code in latest commit.

Reply:
The text history is being created, but the dispatch is not working.

Reply:
It's mistake. Fix it please.